model/raw/src/main/scala/aqua/raw/value/ValueRaw.scala

```diff
@@ -188,6 +207,18 @@ case class CollectionRaw(
     copy(values = values.map(_.renameVars(map)))
 }
 
+object CollectionRaw {
+  def mapCollection[F[_]: Functor](f: ValueRaw => ValueRaw, values: F[ValueRaw]): (F[ValueRaw], F[CollectibleType]) = {
```

Comment:
Maybe lets add call to `CollectionType.elementTypeOf` here?
So that this function returns `(F[ValueRaw], CollectibleType)`